lib/Controllers/BlogController.php

```diff
@@ -19,6 +20,13 @@ public function __construct(BlogPostRepository $blogPostRepository)
     }
 
     public function index(SourceFile $sourceFile) : ControllerResult
+    {
+        return new ControllerResult([
+            'blogPosts' => array_slice($this->blogPostRepository->findAll(), 0, 10),
```

Comment:
We should put this slicing into the repository with an limit argument.

Reply:
Done.